webpack.config.js

```diff
@@ -246,6 +246,7 @@ var config = {
         fallback: {
             crypto: require.resolve('crypto-browserify'),
             stream: require.resolve('stream-browserify'),
+            buffer: require.resolve('buffer/'),
```

Comment:
Is that slash supposed to be there?

Reply:
The forward slash tells node.js module to lookup node_modules instead of core
Reference https://www.npmjs.com/package/buffer#usage